dpgen/generator/arginfo.py

```diff
@@ -41,6 +42,8 @@ def data_args() -> List[Argument]:
     doc_sys_batch_size = 'Each number is the batch_size for training of corresponding system in sys_configs. If set to auto, batch size will be 32 divided by number of atoms.'
 
     return [
+        Argument('init_multi_systems', bool, optional=True,
```

Comment:
`init_multi_systems` has been deprecated in #730.

Reply:
init_multi_systems still work in dpgen run, why deleted.